src/Plugin/Field/FieldFormatter/CaptureUtilityMapFormatter.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Drupal\web_page_archive\Plugin\Field\FieldFormatter;
+
+use Drupal\Component\Utility\Html;
+use Drupal\Core\Field\FieldItemInterface;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Field\FormatterBase;
+
+/**
+ * Plugin implementation of the 'web_page_archive_capture_formatter' formatter.
+ *
+ * @FieldFormatter(
+ *   id = "web_page_archive_capture_utility_map_formatter",
+ *   label = @Translation("Web page archive capture utility map formatter"),
+ *   field_types = {
+ *     "map"
+ *   }
+ * )
+ */
+class CaptureUtilityMapFormatter extends FormatterBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function viewElements(FieldItemListInterface $items, $langcode) {
+    $elements = [];
+    foreach ($items as $delta => $item) {
+      $elements[$delta] = ['#markup' => $this->viewValue($item)];
+    }
+
+    return $elements;
+  }
+
+  /**
+   * Generate the output appropriate for one field item.
+   *
+   * @param \Drupal\Core\Field\FieldItemInterface $item
+   *   One field item.
+   *
+   * @return string
+   *   The textual output generated.
+   */
+  protected function viewValue(FieldItemInterface $item) {
+    return nl2br(Html::escape(implode("\n", array_values($item->toArray()))));
```

Comment:
But really ... this is just how I'm interpreting it, I'll always look to Drupal core for how to implement things, and there's not really a similar example anywhere to check with.